scripts/3.3.0/vagrant_install_simple/script.sh

```diff
@@ -71,8 +72,16 @@ cfy bootstrap -v -p $BLUEPRINT_FILE  -i $INPUTS_FILE --install-plugins --keep-up
 #sleep 10 # wait for ui to start
 #echo finished installing cloudify-ui
 
-
+# cfy blueprints publish-archive -l https://github.com/cloudify-cosmo/cloudify-nodecellar-example/archive/master.tar.gz -b nodecellar1 -n simple-blueprint.yaml
 
 if [ "$INSTALL_SYSTEM_TESTS_REQ" = "true" ]; then
+
+    echo "host_ip: '10.10.1.10'
```

Comment:
Yes, you are right. I've added installation for future testing, but now it is not necessary.